pipeline/ffmpeg.go

```diff
@@ -125,9 +125,6 @@ func (f *ffmpeg) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 
 	job.transcodedSegments = transcodedSegments
 
-	// Transcoding Finished
-	job.ReportProgress(clients.TranscodeStatusCompleted, 1)
```

Comment:
Hmm nice catch. Maybe what we wanted here was progress 1 but status just "transcoding"?

Reply:
Yeah, I think that will actually happen inside the Transcode loop already though. This was really just a leftover from me PoC-ing the ffmpeg pipeline